### PR TITLE
cross-post mails rather than sending a separate mail to each list

### PR DESCRIPTION
**Pull request by tstellar**

### CODE REVIEW

File: AWS/Lambda/PREmailer/pr_emailer.py [hunk 1]
```diff
@@ -388,56 +388,47 @@ def lambda_handler(event, context):
...
-        if event_kind == 'pull_request' and action == 'opened':
-            message_id = first_message_id
-        else:
-            in_reply_to = first_message_id
+    sorted_project_emails = sorted([project_path_email[p] for p in project_list])
+    mail_to = ",".join(sorted_project_emails)
+    if not is_main_branch_event(event, gh_pr):
+        mail_to = LLVM_BRANCH_COMMITS_ADDRESS
+
+    first_message_id = f'<llvm/llvm-project/pull/{pr_number}@llvm.org>'
```
Comment (mstorsjo) [inline AWS/Lambda/PREmailer/pr_emailer.py:400]: Isn't this a duplicate of line 414 below, or is there something I'm missing?

**Review by mstorsjo — COMMENTED** [review]
Looks great overall, thanks! One minor comment inline.

**Review by mstorsjo — APPROVED** [review]
LGTM, thanks! (No idea if this actually works like this, or if the `mail_to` for multiple recipients need to be handled differently etc - but I assume you've looked that up.)

**Comment by tstellar** [comment]
I'll try to deploy this tomorrow.

File: AWS/Lambda/PREmailer/pr_emailer.py [hunk 2]
```diff
@@ -386,56 +386,45 @@ def lambda_handler(event, context):
...
-            mail_to = LLVM_BRANCH_COMMITS_ADDRESS
-
-        # If we're sending an additional email to the same address, break instead
-        if mail_to == last_mail_to:
-            break
-
-        if 'MAIL_TO' in os.environ:
-            mail_to = os.environ.get('MAIL_TO')
-        # Setup the mail Subject
-        subject = f"[{project}] {pr_title} (PR #{pr_number})"
```
Comment (tstellar) [inline AWS/Lambda/PREmailer/pr_emailer.py:411]: This patch inadvertently dropped the project tag from the beginning of the subject, so I've rolled back to a previous version.  I'll send another PR to fix this.

**Pull request merged**